testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/openshift/OpenShiftTokenReviewEndpointTest.java

```diff
@@ -228,29 +241,10 @@ public void customScopes() {
     }
 
     @Test
-    public void emptyScope() throws Exception {
```

Comment:
Why to remove this test? Is it not valid anymore?

Reply:
Yes, this is not valid anymore. If you remove all the scopes, you won't get any audience in the token. So with adding audience to the Token Retrospection Endpoint, this has no sense anymore.